packages/corvid-cli/src/apps/open-editor.js

```diff
@@ -20,6 +20,11 @@ const openEditorApp = ({ useSsl = true } = {}) => ({
         console.log(message);
       });
 
+      client.on("fatal-error", message => {
+        logger.warn(`Fatal Error ${message}`);
```

Comment:
In a scenario which:
1. The user changed a .wix file
2. The .wix file **is** parse-able
Will the editor report a fatal error? What will be the message the user sees?

Reply:
if user had changed .wix file in middle of a session editor will ask him to refresh and after refresh in will close electron and write parse error in cli

Reply:
if .wix file is parse-able then electron will not close of course